### DISCUSSION TITLE
MongoDB Compatibility

### DISCUSSION THREAD

**Discussion by Caligula91** [Automattic/mongoose#13097] [Q&A] [answered]
Is mongoose 7 compatible with mongoDB 4.0.x or 4.2.x?

**Comment by vkarpov15** [comment, marked as answer]
We need to add this data to our compatibility table. But yes, Mongoose 7 uses MongoDB Node driver 5, which supports all the way back to MongoDB server 3.6: https://www.mongodb.com/docs/drivers/node/current/compatibility/

**Reply by hasezoey** [reply]
for future notice, mongoose-mongodb(server) compatibility can be found at https://mongoosejs.com/docs/compatibility.html
